api/message.go

```diff
@@ -129,7 +129,12 @@ func (a *MessageAPI) DeleteMessage(ctx *gin.Context) {
 func (a *MessageAPI) CreateMessage(ctx *gin.Context) {
 	message := model.Message{}
 	if err := ctx.Bind(&message); err == nil {
-		message.ApplicationID = a.DB.GetApplicationByToken(auth.GetTokenID(ctx)).ID
+		application := a.DB.GetApplicationByToken(auth.GetTokenID(ctx))
+		message.ApplicationID = application.ID
+		// if title is empty set application.name as default value
```

Comment:
This comment doesn't really add something to the code, the code speaks for it self, so please remove it.